### PR TITLE
Update DG and add PPP 

### PR DESCRIPTION
**Pull request by jodichoo**
- Appendix C updated
- Typos in DG fixed
- Terms in sequence diagrams standardized (commandResult)
- Added PPP 
- Minor edits to UG for clarity

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -959,21 +957,35 @@ Expected: No module is added. Error details are shown in the status message.
...
 
 1. Modifying all AB3 components
 
-   We need to modify the entire AB3 in order to support operations on modules.
-   In `Model` component, we need to delete all models in the `person` folder and create a new folder that contains the model for module and all other attributes related to module(`Code`, `Title`, `Description`, `Mc`, ). All other classes in `model` folder need to be modified accordingly. 
-   In `Logic` component, we need to modify all command parsers in order to parser modules. All commands also need to be modified to support the operation on modules.
-   In `storage` component, we also need to modify all classes in order to store modules.
-   The `UI` component also need to be modified. 
-   All test cases need to be changed accordingly.
+   We needed to modify the entire AB3 in order to support operations on modules.
```
Comment (wengYing227) [inline docs/DeveloperGuide.md:966]: `AB3` should be changed to `NUS Mod Tracker`

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -959,21 +957,35 @@ Expected: No module is added. Error details are shown in the status message.
 Expected: Similar to previous.
 
 
-## **Appendix C: Instructions for manual testing**
+## **Appendix C: Effort**
 
-If the effort required to create AB3 is 100, we would place the effort required to implement the current version of NUS Mod Tracker at 120. We currently has over 10,000 lines of code contributed and over 300 automated tests. Below, we list some changes that need a significant amount of effort to implement.
+If the effort required to create AB3 is 100, we would place the effort required to implement the current version of NUS Mod Tracker at 120. Our team has contributed over 10,000 lines of code contributed and over 300 automated tests. Below are some changes in particular, that required a larger amount of effort to implement.
 
 1. Modifying all AB3 components
```
Comment (wengYing227) [inline docs/DeveloperGuide.md:964]: `AB3` hasn't been changed

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -959,21 +957,35 @@ Expected: No module is added. Error details are shown in the status message.
...
 
-   We need to modify the entire AB3 in order to support operations on modules.
-   In `Model` component, we need to delete all models in the `person` folder and create a new folder that contains the model for module and all other attributes related to module(`Code`, `Title`, `Description`, `Mc`, ). All other classes in `model` folder need to be modified accordingly. 
-   In `Logic` component, we need to modify all command parsers in order to parser modules. All commands also need to be modified to support the operation on modules.
-   In `storage` component, we also need to modify all classes in order to store modules.
-   The `UI` component also need to be modified. 
-   All test cases need to be changed accordingly.
+   We needed to modify the entire AB3 in order to support operations on modules.
+
+   In the `Model` component, we needed to update all models in the `person` folder with a new folder that contains the model for module, as well as all other attributes related to a module (`Code`, `Title`, `Description`, `Mc`, ).
```
Comment (wengYing227) [inline docs/DeveloperGuide.md:968]: We don't have `person` folder

File: docs/UserGuide.md [hunk 4]
```diff
@@ -241,12 +241,13 @@ Tip: Do take a good look at this section to understand the common terminologies
 #### `CODE`
 * Represents the `module`'s code component.
 * `CODE` must follow the NUSMods module code format.
-* `CODE` should start with 2-3 capital letters, followed by 4 numerical digits, and end with an optional capital letter.
-* Valid Examples: CJ1101K, CH1001, NMB1234
-* Invalid Examples: CA300M, GEY10005, G1234M
+  * `CODE` should start with 2-3 capital letters, followed by 4 numerical digits, and end with an optional capital letter.
+  * Valid Examples: CJ1101K, CH1001, NMB1234
```
Comment (wengYing227) [inline docs/UserGuide.md:245]: How about adding `UIS3911EL` as a valid example?

Comment (wengYing227) [reply docs/UserGuide.md:245]: But then we also need to modify our code

**Review by wengYing227 — COMMENTED** [review]
Overall looks good, but there are still some minor issues that need fixing

File: docs/UserGuide.md (hunk 4 above)
Comment (jodichoo) [reply docs/UserGuide.md]: Sure, its a valid module so I have made the changes 👍

**Review by LuoZhijie-tom — APPROVED** [review]
LGTM

**Pull request merged**